support-frontend/stories/checkoutLayout/CheckoutBenefitsList.stories.tsx

```diff
@@ -45,7 +45,7 @@ export const AllBenefitsUnlocked = Template.bind({});
 
 AllBenefitsUnlocked.args = {
 	title: "For £12 per month, you'll unlock",
-	checkListData: checkListData({ higherTier: true }),
+	checkListData: checkListData({ higherTier: true },true),
```

Comment:
Do you use prettier? Its very good add-in for visual-studio. It's just that it complains if you don't put a space before variables (e.g checkListData: checkListData({ higherTier: true }, true)) ie : space before the final true. As say just tiny. tiny thing.

Reply:
Oh really, did not know that.I used IntelliJ, not Visual Studio.So it was not flagged there.Have added the space.Thanks